metaflow/linters/pylint/pylint_wrapper.py

```diff
@@ -0,0 +1,124 @@
+import re
+import sys
+
+from metaflow.cmd.configure_cmd import echo, echo_always
+from metaflow.exception import MetaflowException
+from metaflow.linters.lint import LintWarn
+from metaflow.linters.linter_interface import LinterWrapperInterface
+
+try:
+    from StringIO import StringIO
+except ImportError:
+    echo("unable to import StringIO, possibly not installed. Resorting to `io`", fg="yellow")
+    from io import StringIO
+except Exception as e:
+    raise MetaflowException(e)
+
+from metaflow.extension_support import get_aliased_modules
+
+
+class PyLintWarn(LintWarn):
+    headline = "Pylint is not happy"
+
+
+class PyLintWrapper(LinterWrapperInterface):
+    def __init__(self):
```

Comment:
PyLintWrapper has simply thence, been lifted and shifted to preserve/support compatibility with this previous and to-be-deprecated choice @savingoyal